docs/data-sources/app_builder_app.md

```diff
@@ -0,0 +1,35 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "datadog_app_builder_app Data Source - terraform-provider-datadog"
+subcategory: ""
+description: |-
+  This data source retrieves the definition of an existing Datadog App from App Builder for use in other resources.
+---
+
+# datadog_app_builder_app (Data Source)
+
+This data source retrieves the definition of an existing Datadog App from App Builder for use in other resources.
```

Comment:
Is there any use case where customers might want to rely on this data source? Might be worth having an example here.

Reply:
Does this suffice?
```text
This data source retrieves the definition of an existing Datadog App from App Builder for use in other resources, such as embedding Apps in Dashboards or Notebooks.
```

Reply:
Does this suffice?
```text
This data source retrieves the definition of an existing Datadog App from App Builder for use in other resources, such as embedding Apps in Dashboards or Notebooks.
```